config/124-elixir-config.el

```diff
@@ -1 +1,6 @@
 (use-package alchemist :ensure t)
+
+(use-package elixir-mode :ensure t
+  :bind (("C-c M-j" . alchemist-iex-project-run)
```

Comment:
These bindings should belong to alchemist shouldn't they ?

Reply:
Addressed. Everything works still. elixir-mode is being used in the hook to format elixir files on save.